client/model-builder/src/main/java/io/smallrye/graphql/client/model/ClientModelBuilder.java

```diff
@@ -75,6 +80,19 @@ private ClientModels generateClientModels() {
         return clientModels;
     }
 
+    private void validateAnnotations(Collection<AnnotationInstance> graphQLApiAnnotations) {
```

Comment:
Could we name this `validateNamespaceAnnotations` to make it more clear what it does?